gptqmodel/utils/vllm.py

```diff
@@ -0,0 +1,24 @@
+
+from vllm import LLM, SamplingParams
+
+def load_model_by_vllm(
+    model,
+    trust_remote_code,
+    **kwargs,
+):
+    model = LLM(
+        model=model,
+        trust_remote_code=trust_remote_code,
```

Comment:
There is no need to explicitly add trust_remove_code here. it is passed, if needed, via **kargs